staging/src/k8s.io/api/admissionregistration/v1/types.go

```diff
@@ -462,8 +462,8 @@ const (
 // RuleWithOperations is a tuple of Operations and Resources. It is recommended to make
 // sure that all the tuple expansions are valid.
 type RuleWithOperations struct {
-	// Operations is the operations the admission hook cares about - CREATE, UPDATE, or *
-	// for all operations.
+	// Operations is the operations the admission hook cares about - CREATE, UPDATE, DELETE, CONNECT or *
+	// for all operations. Here "all" means "all above four admission operations".
```

Comment:
```suggestion
	// Operations is the operations the admission hook cares about - CREATE, UPDATE, DELETE, CONNECT or *
	// for all of those operations and any future admission operations that are added.
```

Reply:
also update the same comment in v1beta1/types.go and run `make update` to update generated docs.